frontends/web/src/locales/de/app.json

```diff
@@ -598,6 +600,28 @@
       "title": "Geräteinformationen"
     },
     "expert": {
+      "bip85": {
+        "description": "Erstelle Unterschlüssel von deiner BitBox Wallet",
```

Comment:
I got some feedback and we should better use English term "Child key" as in BIP0085 also in German.

We do use English terms in German (and other langs) such as "Coin Control", "Full node" or "hardware wallet".

I'll try to ping the translator.